Step1/main.tf

```diff
@@ -0,0 +1,46 @@
+# main.tf
+
+provider "aws" {
+  region = var.aws_region
+}
+
+resource "aws_instance" "windows_instance" {
+  ami           = var.windows_ami_id

```

Comment:
What if I don't want to care about the AMI ID?

Reply:
**Fixed.** Terraform will dynamically select the latest available Windows AMI, to ensures that the instance will be built with the most current AMI, reducing the need for manual updates.